pkg/cmd/docgen/diagrams.go

```diff
@@ -1137,8 +1143,10 @@ var specs = []stmtSpec{
 	},
 	{
 		name:  "show_tables",
-		stmt:  "show_stmt",
-		match: []*regexp.Regexp{regexp.MustCompile("'SHOW' 'TABLES'")},
+		stmt:  "show_tables_stmt",
+		inline:  []string{"with_comment"},
+		replace: map[string]string{"name": "database","'.' name": "'.' schema"},
```

Comment:
I think that this won't work sometimes because the order it will happen is random. That is, sometimes the `'.' name` will get replaced first, and sometimes the `name` will, causing problems. Can you add something before the first `name` to make it unambiguous from the other one?